tests/unit/changeDir.ts

```diff
@@ -0,0 +1,22 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import changeDir from 'src/changeDir';
+import { stub, SinonStub } from 'sinon';
+
+let changeDirStub: SinonStub;
+
+registerSuite({
+	name: 'changeDir',
+	'setup'() {
+		changeDirStub = stub(process, 'chdir');
```

Comment:
I meant:

`process.ts`:

```ts
export default process;
```

And then `changeDir.ts`:

```ts
import process from './process';
```

That way you can stub the `./process` import without having to stub the actual `process` object.